docs/static_site/src/assets/js/options.js

```diff
@@ -66,8 +65,10 @@ $(document).ready(function () {
         $('button.opt').each(function(){is_a_match($(this), environSelect)});
 
         showContent();
+        let queryString = '?version=' + versionSelect + '&platform=' + platformSelect + '&language=' + languageSelect + '&environ=' + environSelect + '&processor=' + processorSelect;
+        window.location.search = queryString;
```

Comment:
this line should go otherwise it triggers a refresh of the page. the history.pushState is sufficient for changing the URL.